api/api/events/models.py

```diff
@@ -0,0 +1,16 @@
+from accounts.models import User
+from clubs.models import Club
+from django.db import models
+
+
+class Events(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+    starts_at = models.DateField()
+    ends_at = models.DateField()
+    registration_required = models.BooleanField(default=False)
+    created_by = models.OneToOneField(User, on_delete=models.PROTECT)
+    image_url = models.URLField(max_length=500)
+    club = models.OneToOneField(Club, on_delete=models.PROTECT)
```

Comment:
Add a comment on top of this: `This club is the primary organizer of this event.`